easy-note/cmd/note/handler.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2022 CloudWeGo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package main
+
+import (
+	"context"
+
+	"easy-note/cmd/note/pack"
+	"easy-note/cmd/note/service"
+	"easy-note/kitex_gen/demonote"
+	"easy-note/pkg/consts"
+	"easy-note/pkg/errno"
+)
+
+// NoteServiceImpl implements the last service interface defined in the IDL.
+type NoteServiceImpl struct{}
+
+// CreateNote implements the NoteServiceImpl interface.
+func (s *NoteServiceImpl) CreateNote(ctx context.Context, req *demonote.CreateNoteRequest) (resp *demonote.CreateNoteResponse, err error) {
+	resp = new(demonote.CreateNoteResponse)
```

Comment:
use https://github.com/cloudwego/thrift-gen-validator

Reply:
All the problems are solved, please check.